### PR TITLE
currency selector UI #199

### PR DESCRIPTION
**Pull request by matjaz**
see #199 
Screen Recording 2024-08-10 at 12 42 03

### CODE REVIEW

File: src/utils/currencies.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+export const getCurrencies = () => {
+  const currencies: {name: string; value: string; flag: string}[] = [
+    {name: 'SATS', value: 'sats', flag: '₿'},
+    {name: 'BTC', value: 'BTC', flag: '₿'},
+    {name: 'USD', value: 'USD', flag: getFlag('US')},
+    {name: 'EUR', value: 'EUR', flag: '🇪🇺'},
+  ];
+
+  const ISOMapping = {
```
Comment (rolznz) [inline src/utils/currencies.ts:16]: where is this from? could you add a comment with a link?

**Comment by rolznz** [comment]
@matjaz nice work!
image

The styling seems broken to me (sometimes the text is on the right, sometimes below), and it's hard to scroll through the list because it's vertically so short. 

Could you check the design here: https://github.com/getAlby/bitcoin-connect/issues/199 - there should be 2 columns and the scrollable area should be taller.

**Comment by matjaz** [comment]
changed grid to 2 columns and added some padding

**Comment by rolznz** [comment]
image
image
image

@stackingsaunter styling is not perfect, but I think it looks good to me given the restrictions/requirement. What do you think?

**Comment by rolznz** [comment]
@matjaz thanks for your work, we will pay out the 150K sats to the lightning address on your github profile.

**Comment by matjaz** [comment]
thank you!

**Pull request merged**